drivers/sensor/tdk/mpu6050/Kconfig

```diff
@@ -69,4 +69,13 @@ config MPU6050_GYRO_FS
 	  An X value for the config represents a range of +/- X degrees/second.
 	  Valid values are 250, 500, 1000, 2000.
 
+config MPU6050_SAMPLE_RATE_DIVIDER
+	int "Data Sample Rate Divider"
+	default 7
```

Comment:
Help indicates that the recommended sensor data sample rate is 1600Hz, why not use that one?

Reply:
It's a typo, should be changed to max recommended sample rate